CHANGELOG.md

```diff
@@ -1,10 +1,66 @@
 # nf-core/tools: Changelog
 
-## v1.7
+## v1.8
 
-### PyPI package description
+### Tools helper code
 
-* The readme should now be rendered properly on PyPI.
+* Refactored the template synchronisation code to be part of the main nf-core tool
+* `nf-core bump-version` now also bumps the version string of the exported conda environment in the Dockerfile
+* Updated Blacklist of synced pipelines
+* Ignore pre-releases in `nf-core list`
+* Updated documentation for `nf-core download`
+* Fixed typo in `nf-core launch` final command
+* Handle missing pipeline descriptions in `nf-core list`
+
+### Linting
+
+* Adjusted linting to enable `patch` branches from being tested
+* Warn if GitHub actions workflows do not exist, warn if `.travis.yml` and circleCI are there
+* Lint for `Singularity` file [and remove it](https://github.com/nf-core/tools/issues/458)
+* Added linting of GitHub actions workflows `linting.yml`, `ci.yml` and `branch.yml`
+* Warn if pipeline name contains upper case letters or non alphabetical characters [#85](https://github.com/nf-core/tools/issues/85)
+
+### Template pipeline
+
+* Fixed incorrect paths in iGenomes config as described in issue [#418](https://github.com/nf-core/tools/issues/418)
+* Fixed [incorrect usage of non-existant parameter](https://github.com/nf-core/tools/issues/446) in the template
+* Add UCSC genomes to `igenomes.config` and add paths to all genome indices
+* Change `maxMultiqcEmailFileSize` parameter to `max_multiqc_email_size`
+* Export conda environment in Docker file [#349](https://github.com/nf-core/tools/issues/349)
+* Change remaining parameters from `camelCase` to `snakeCase` [#39](https://github.com/nf-core/hic/issues/39)
```

Comment:
```suggestion
* Change remaining parameters from `camelCase` to `snake_case` [#39](https://github.com/nf-core/hic/issues/39)
```
if it's `snakeCase` I can't see the difference with `camelCase`

Reply:
Good spot 👍 I changed that already in https://github.com/nf-core/tools/pull/483